frontend/tablet/src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { StyleSheet, Text } from 'react-native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+
+interface Props {
+  title: string
+  onPress: () => void
+}
+
+const Button: React.FC<Props> = (props) => {
```

Comment:
You can use 
```
const Button: React.FC<ButtonProps> = ({ title, onPress }) => {
```
to destructure the `props` and then you can use just `title` instead of `props.title` and so on.

Reply:
All requested changes have been modified in new commits.